builder/proxmox/iso/step_download_iso_on_pve.go

```diff
@@ -5,44 +5,32 @@ package proxmoxiso
 
 import (
 	"context"
-	"fmt"
 
-	"github.com/Telmate/proxmox-api-go/proxmox"
+	proxmoxcommon "github.com/hashicorp/packer-plugin-proxmox/builder/proxmox/common"
 	"github.com/hashicorp/packer-plugin-sdk/multistep"
-	packersdk "github.com/hashicorp/packer-plugin-sdk/packer"
 )
 
 // stepDownloadISOOnPVE downloads an ISO file directly to the specified PVE node.
 // Checksums are also calculated and compared on the PVE node, not by Packer.
-type stepDownloadISOOnPVE struct{}
+type stepDownloadISOOnPVE struct {
```

Comment:
Is there a reason to keep this step in the code since we're defining a `common` step for this purpose? I would think the `common` implementation supersedes this one, and therefore we could remove this code.

Reply:
The problem is that the file path is returned in two different ways. In `common` I can return it via the `additionalISOsConfig` struct. But in `iso` I can only get a string variable, so it was solved by putting it in the `iso-config` state bag.
It is possible to move that logic to `stepUploadISO` and 'stepUploadAdditionalISO' and reuse the `DownloadISOOnPVE` function there. I don't know if this is a better solution.